include/termcolor/termcolor.hpp

```diff
@@ -53,214 +52,264 @@ namespace termcolor
     {
         inline int colorize_index();
         inline FILE* get_standard_stream(const std::ostream& stream);
-        inline bool is_colorized(std::ostream& stream);
-        inline bool is_atty(const std::ostream& stream);
+        inline FILE* get_standard_stream(const std::wostream& stream);
```

Comment:
I tried, but here's what happens
![image](https://user-images.githubusercontent.com/18056374/145793767-b85130b4-2a4f-4a01-85b9-8014cffb48b6.png)
![image](https://user-images.githubusercontent.com/18056374/145793810-7109b667-186f-4ea7-a8d1-9d2e5454d9a2.png)

Reply:
It needs C++17>

Reply:
Well, we can probably `reinterpret_cast<const char*>`  them since we don't care about types here. Also, with everything else fixed, it does not look that bad having just these two specializations for `get_standard_stream`.